doc/API/centreon-api-v2.yaml

```diff
@@ -183,13 +183,46 @@ paths:
 
         The available parameters to **search** / **sort_by** are:
 
+        * id
+        * name
+        * type
+        * status_code
+        * status
+        * action_url
+        * details_url
+        * parent_name
+        * parent_status
+        * severity_level
+        * in_downtime
+        * acknowledged
+        * impacted_resources_count
+        * last_status_change
+        * tries
+        * last_check
+        * information
+        * host.group
```

Comment:
not sure we'll allow to order by host group

Reply:
why we have to restrict if it's possible?

Reply:
because, for instance, a BA is not linked to a hostgroup